src/deployment.ts

```diff
@@ -152,12 +152,23 @@ export class Deployment {
 
             disposables.push(inputBox.onDidHide(() => resolve(null)));
 
-            disposables.push(inputBox.onDidChangeValue((e) => {
+            disposables.push(inputBox.onDidChangeValue(async (e) => {
                 if (validator.isURL(inputBox.value)) {
                     inputBox.validationMessage = undefined;
                 } else {
                     inputBox.validationMessage = 'Please enter a valid URL';
                 }
+
+                //check the mentioned url has image referece
+                inputBox.validationMessage = 'Checking the image info'
+                inputBox.enabled = false;
+                if (! await Oc.Instance.hasImageInfo(inputBox.value)) {
```

Comment:
This makes the validation SO SLOW, so that it misses the characters I'm typing in.


https://github.com/user-attachments/assets/9a15e609-f8c0-4218-8c7c-63723a224f70

The text I've typed in was: "docker.io"